graylog2-web-interface/src/components/events/events/EventDetails.jsx

```diff
@@ -94,13 +113,16 @@ class EventDetails extends React.Component {
               &emsp;
               ({plugin.displayName || event.event_definition_type})
             </dd>
+            <dd>
+              {this.renderReplaySearchLink(event)}
```

Comment:
I think we could wrap this in a container with an `Actions` title. Currently, the `Replay search` link is right below the link to the event definition, which makes it a bit hard to distinguish.